tests/probe/t14-specimen.sh

```
@@ -0,0 +1,23 @@
+# branch.
+
+collect_logs() {
+  DIR=$1
+  cd $DIR
+  for f in `ls`; do
+    if [ $f == "skip" ]; then
+      continue
+    fi
+    cat $f | grep "ERROR" > /tmp/errors.txt
+  done
+  echo "wrote " $(cat /tmp/errors.txt | wc -l) " lines"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)